### ISSUE TITLE
model-zoo Cifar10.jl is generating "Loss is NaN"

### ISSUE THREAD

**Issue by caxelrud** [FluxML/Flux.jl#970]
model-zoo Cifar10.jl (commit Oct/1) is generating the following results when running this code line: 

> Flux.train!(loss, params(m), train, opt, cb = evalcb)
> accuracy(valX, valY) = 0.119
> ERROR: Loss is NaN

Versions: Flux v0.8.3, Julia v1.1.1, Windows 10, GPU: GeForce GTX 1660 Ti

**Comment by mrchaos** [comment]
see https://github.com/mrchaos/model-zoo/blob/master/vision/cifar10/cifar10_gpu_minibatch.jl

**Comment by CarloLucibello** [comment]
closing this as it pertains to model-zoo. Possibly fixed by using logitcrossentropy insead of sofmax+crossentropy
